lib/model/frames/submission.js

```diff
@@ -24,7 +24,9 @@ class Submission extends Frame.define(
   'instanceId', readable,               'submitterId',  readable,
   'deviceId',   readable,               'createdAt',    readable,
   'updatedAt',  readable,               'reviewState',  readable, writable,
-  embedded('submitter')
+  'userAgent',  readable,
```

Comment:
Since we didn't return `userAgent` on the logical submission before, could you add a test that verifies that this field is now returned? And that it's specifically the `userAgent` of the root version?